Composer/packages/lib/code-editor/src/BaseEditor.tsx

```diff
@@ -23,6 +23,7 @@ const defaultOptions: monacoEditor.editor.IEditorConstructionOptions = {
   glyphMargin: false,
   folding: false,
   renderLineHighlight: 'none',
+  formatOnType: true,
```

Comment:
Let's make sure this doesn't cause weirdness in the other places that the code-editor is used.

Reply:
All editor instances inherited from baseEditor are enabled for document formating command as a client. But here only in LG editor will have the ability to do so since the LG server send 
      documentOnTypeFormattingProvider: {
                 firstTriggerCharacter: '\n',
              }
at initialization. 
Moreover, the code at server side will make sure the  documentOnTypeFormatting will only affect typing in structure LG.

I will test all other editor.

Reply:
> Let's make sure this doesn't cause weirdness in the other places that the code-editor is used.

I tested the editors in composer. Only the LG editor will have the ability to format structure lg input.

Reply:
thanks for investigating @cosmicshuai